content/pieces/logging-off/notes/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Logging Off
+author: Celine Nguyen
+layout: multipage
+order: celine
+# custom frontmatter
+---
+
+### Foraging for mushrooms
+
+If you are interested in mushroom foraging, there are a number of regional mycological societies that do mushroom walks. It’s very likely that the protagonist of this story would have consulted information from organizations like the [Bay Area Mycological Society](http://bayareamushrooms.org/) and [Mycological Society of San Francisco](https://www.mssf.org/).
+
+### Foraging for fruit
```

Comment:
@maisutton this part wasn't in the Google Doc (realised I forgot to link the Falling Fruit site!) so wanted to flag it for you